tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
Sure, I am all for new tests that test different functionality but you have a test_GetIpParents and then there is another test that already exists called testGetIpParents. These look to be exactly the same. Duplicate tests don't really help.

The following look to be duplicates of existing tests
test_GetIpParents
test_GetServiceParent
test_GetNetParent
test_GetService
test_GetService_raises_if_unknown
test_GetServiceByProto
test_GetNet
test_GetNet_raises_if_unknown
test_parseServiceList

These both looks to be unneeded as they only call another test.
test_GetNetAddr
test_GetNetAddr_raises_if_unknown

The other ones should be moved into the normal testing class. In your comment you admit that these are not really characterization tests.